### DISCUSSION TITLE
MIS IOTA HAN DESAPARECIDO AL PINCHAR EN "GANASTE 2000 IOTA... HAGA CLIC..."

### DISCUSSION THREAD

**Discussion by 19801980David** [iotaledger/firefly#8642] [Help & Support]
HOLA, en mi cartera de Firdfly me aparecian correctamente mis 2038 IOTA, después de haber HECHO CLICK EN EL ANUNCIO ESTAFA DE: "YOU WON 2000 IOTA... CLICK... de Iota 1q...2xht7a...(el cual adjunto pantallazo) me quedaron 0,23975 iota!! 
Con fecha 4 de Octubre del 2023, aparece una transacción de 2003,6635 iotas a una dirección desconocida, que yo NO hice dicha transferencia. 
 ¿Puedo informarme de lo sucedido? 
 Pueden ayudarme a recuperar mis 2038 IOTA. Muchas gracias, si necesitan mas información no tengo problema en facilitarsela.
Gracias

**Comment by begonaalvarezd** [comment]
Siento mucho que hayas sido victima del scam de lotería, nunca deberías ingresar su mnemonic en ninguna web o aplicacion no oficial. Me temo que si la transaccion se ha hecho sin que tu la hayas autorizado, los scammers te han robado

Hay estafadores que están enviando NFTs a direcciones de IOTA con el objetivo de engañar a los usuarios para que visiten una web de "Lotería de IOTA" (IOTA Lottery) donde se les promete que pueden reclamar su "Premio".

No existe ninguna Lotería de IOTA, y visitar este sitio web puede llevar a perder todos los fondos en su billetera/cartera de IOTA.

No es posible bloquear la recepción de estos NFT, así que simplemente ignóralos o "quémalos" siguiendo estos simples pasos:
En tu billetera/cartera ve a:

- Coleccionables
- Selecciona el NFT del estafador
- Haz clic en los tres puntos en la parte superior derecha
- Quemar NFT

NUNCA ingreses tu semilla, frase de 24 palabras o tu backup de stronghold en NINGÚN sitio web o aplicación.

Mantente seguro.

Si fuiste víctima de esta estafa, por favor repórtala a las autoridades locales.

**Reply by begonaalvarezd** [reply]
Firefly no te deja hacer click en el enlace, has debido copiar en enlace y pegarlo en un navegador, si ingresaste tu mnemonic en la web del scammer, desgraciadamente le has dado tu mnemonic al scammer. De nuevo, siento mucho que hayas sido victima de la estafa 🙏🏼 

Si por el contrario no ingresaste tu mnemonic en ninguna web, comprueba que la transaccion efectivamente no es tuya, a veces los usuarios se mandan los fondos a exhanges y no se acuerdan

**Comment by 19801980David** [comment]
Hola Begoña, pues el enlace estaba dentro del mismo programa de Firefly... si te deja hacer click en el enlace, y no copié ningún enlace en el navegador, yo no ingresé mnemonic en la web del scammer, simplemente pinché en el enlace de la lotería de los 2000 Iota desde Firefly.
Les adjunto los pantallazos descritos anteriormente, así como la dirección donde se retiraros los 2003,6635 Iota, la cual le confirmo que no es ninguna dirección de las mias de ningún exchange, ojalá así fuera.
Un saludo y espero una solución si es posible.
![20241216_180232](https://github.com/user-attachments/assets/f0a53303-42ae-46b4-a5c1-0d3767d125e7)
![20241216_173735](https://github.com/user-attachments/assets/de23b1e7-abbb-4fc1-a6df-651e8b00d68a)
![20241216_173949](https://github.com/user-attachments/assets/508d6eaf-005a-4a19-8e07-1f8a6b5f45c6)
![20241216_180726](https://github.com/user-attachments/assets/829a032a-8f93-4105-9b8f-952f6d37cf6b)
![20241216_174114](https://github.com/user-attachments/assets/2e1ee3ad-b78a-41bd-9703-28db14af76fe)

**Reply by begonaalvarezd** [reply]
Si no has visitado la web del scammer y simplemente has recibido el NFT del scammer entonces no te han podido robar por ahí, recibir un NFT, aunque sea un scam, no supone ningun riesgo. El NFT lo puedes quemar o quedartelo, como prefieras. Mientras no se comprometa tu mnemonic nadie puede robarte las claves, te recomiendo que compruebes que la transaccion no es tuya. 

Si tu mnemonic ha sido comprometida por algún otro motivo nosotros no podemos hacer nada, los usuarios son dueños de sus propias claves y tienen que asegurarse que estan seguras, Firefly es una wallet de custodia propia. 

Si necesitas mas ayuda de recomiendo que visites nuestro discord https://discord.iota.org/ y vayas a un canal de ayuda.
